omod-common/src/main/java/org/openmrs/module/webservices/rest/web/response/ObjectNotFoundException.java

```diff
@@ -9,6 +9,8 @@
  */
 package org.openmrs.module.webservices.rest.web.response;
 
+import java.util.Date;
```

Comment:
What are the changes in this file for?

Reply:
I want to rather use a customized constructor.